src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(result, number) {
+      return result + number;
+    },
+
+    subtract(result, number) {
+      return result - number;
+    },
```

Comment:
```suggestion
    subtract: (result, number) => result - number;
```
you can make it simplier